### ISSUE TITLE
Work wiht ACR1222L

### ISSUE THREAD

**Issue by nikomateos** [MorningCoffeeDev/ACR-NFC-Reader-PhoneGap-Plugin#2]
Hi, the device acr1222l works with this plugin???, Do you have any example. Thanks

**Comment by y00rb** [comment]
yes, it works on ARC1222, you can follow the guide to install and use it.

**Comment by nikomateos** [comment]
Hi y00r!!! Great, works fine!!! But the time of reading is too long, I don´t see the polling time in your Plugin!! Can i change this time in somewhere?? Thanks for all!!

**Comment by y00rb** [comment]
yes, please go ahead, any feedback or ideas, you can contact again. we can help you to make better in plugin.

**Comment by villenero** [comment]
Dear y00rb,

As told my colleague Niko above we're happy with your plugin but the response time that we get with the ACR1222L is quite long (5 secs on every tap).

I was wondering if you can help us optimizing the response time with your expertise. Obviously we can arrange an economical deal.

Please let me know if you're interested, my email is carlos@telemaco.es

Best regards.
